test/xml/TestConnector3.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
```

Comment:
@alexcamposruiz Had to add this file since the issue only occurs when there is no participant_qos.

Reply:
Simplify the file. No need for qos_library, define a dummy type, and only 1 participant.